src/utils/mapUtils.ts

```
@@ -1,13 +1,30 @@
 import { RefObject } from 'react'
+import { LayerInfo } from '../types/MapDataTypes'
 
 export function getActiveLayers(mapLayers: LayerInfo[]): string[] {
   return mapLayers.filter((layer) => layer.isLayerOn).map((layer) => layer.layerId)
 }
 
+export const applyOpacityExpression = (array) => {
+  if (!Array.isArray(array) || array.length === 0) {
+    return 0
+  }
+  debugger
```

Biome (2.1.2)

[error] This is an unexpected use of the debugger statement.

Unsafe fix: Remove debugger statement


(lint/suspicious/noDebugger)